cluster/gce/gci/configure-helper.sh

```diff
@@ -2089,10 +2089,14 @@ function start-fluentd-resource-update {
   wait-for-apiserver-and-update-fluentd &
 }
 
-# Update {{ container-runtime }} with actual container runtime name.
+# Update {{ container-runtime }} with actual container runtime name,
+# and {{ container-runtime-endpoint }} with actual container runtime
+# endpoint.
 function update-container-runtime {
-  local -r configmap_yaml="$1"
-  sed -i -e "s@{{ *container_runtime *}}@${CONTAINER_RUNTIME_NAME:-docker}@g" "${configmap_yaml}"
+  local -r file="$1"
+  sed -i -e "s@{{ *container_runtime *}}@${CONTAINER_RUNTIME_NAME:-docker}@g" "${file}"
+  container_runtime_endpoint=${CONTAINER_RUNTIME_ENDPOINT:-"unix:///var/run/dockershim.sock"}
```

Comment:
nitpick: Remove the quotes, this is enough:

```shell
local -r container_runtime_endpoint=${CONTAINER_RUNTIME_ENDPOINT:-unix:///var/run/dockershim.sock}
```

Reply:
Done.